### PR TITLE
Distribute everything required to build the docs

### PR DESCRIPTION
**Pull request by ghisvail**
Which includes everything under `docs/source` and the Makefiles under `docs`.

### CODE REVIEW

**Comment by mrocklin** [comment]
This seems fine to me.  I'll leave this up for a bit so that others who are more familiar with sphinx can have an opportunity to double check.

Thanks @ghisvail !

File: MANIFEST.in [hunk 1]
```diff
@@ -1,11 +1,11 @@
 recursive-include dask *.py
-recursive-include docs *.rst
+recursive-include docs/source *
+include docs/Makefile docs/make.bat
```
Comment (TomAugspurger) [inline MANIFEST.in]: Maybe also include `docs/README.rst` and `requirements-docs.txt`?

Comment (ghisvail) [inline MANIFEST.in]: These files aren't part of the nominal set of source files required to generate the Sphinx documentation. Should they be included, it would be the subject of a different PR.

**Comment by mrocklin** [comment]
I'm just going to go ahead and merge this.  It seems like an improvement.  Thanks @ghisvail

**Comment by mrocklin** [comment]
I would expect a release sometime in the next couple of weeks.  I'll ask
this of the other developers at a meeting today.
